5.15.2 (#5761)
mui/mui-x: CHANGELOG.md

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

> ## 5.16.0

Reviewer: I thought it would be a patch. The quick filtering logic modification is kind of a bug fix

Author: I have to agree on that, even though there are certain changes in "API", but it's not related to `DataGrid`
For example: the `pickers` defaulting to `desktop` mode in tests, but those have their own `beta` version. 👍

> ## 5.16.0

Reviewer 2: ```suggestion
## 5.16.0

```

_Aug 11, 2022_

We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:

- ✨ Improve quick filtering with row grouping (#5701) @alexfauquette
- 📚 Documentation improvements

> - 🐞 Bugfixes

Reviewer 3: Did not noticed that before but sometime we write `Bug fixes` and sometime `Bugfixes`
I personally prefer the format but let's pick one and stick to it

Author: 32 instances of `bugfixes` and 5 instances of `bug fixes`. 🤔 
I guess it's safe to say that we want to `stick` with `bugfixes`? 😄

### `@mui/x-data-grid@5.16.0` / `@mui/x-data-grid-pro@5.16.0` / `@mui/x-data-grid-premium@5.16.0`

#### Changes

> - [DataGrid] Catch error if rows freezing is not supported (#5711) @cherniavskii

Reviewer 4: ```suggestion
- [DataGrid] Catch errors if rows freezing is not supported (#5711) @cherniavskii
```

- [DataGrid] Preserve cell mode when entering edit mode while commiting (#5686) @m4theushw
- [DataGridPremium] Let quick filter search in row grouping children (#5701) @alexfauquette

### `@mui/x-date-pickers@v5.0.0-beta.5` / `@mui/x-date-picker-pro@5.0.0-beta.5`

#### Changes

- [time picker] set clock focus outline to `none` (#5758) @LukasTy

> - [pickers] Fix theme augmentation with typescript (#5596) @alexfauquette

Reviewer 2: ```suggestion
- [pickers] Fix theme augmentation with TypeScript (#5596) @alexfauquette
```

- [pickers] Reset input value when locale is modified (#5310) @alexfauquette

> - [pickers] Support `disableHighlightToday` on MonthPicker and YearPicker (#5562) @flaviendelangle

Reviewer 3: ```suggestion
- [pickers] Support `disableHighlightToday` on `MonthPicker` and `YearPicker` (#5562) @flaviendelangle
```

> - [pickers] Support `disableHighlightToday` on MonthPicker and YearPicker (#5562) @flaviendelangle

Reviewer: ```suggestion
- [pickers] Support `disableHighlightToday` on `MonthPicker` and `YearPicker` (#5562) @flaviendelangle
```

> - [pickers] fix to default to desktop mode when matchMedia is unavailable (#5684) @LukasTy

Reviewer 3: ```suggestion
- [pickers] Fallback to desktop mode when `matchMedia` is unavailable (#5684) @LukasTy
```

> - [pickers] fix to default to desktop mode when matchMedia is unavailable (#5684) @LukasTy

Reviewer: ```suggestion
- [pickers] fix to default mode to `'desktop'` when `matchMedia` is unavailable (#5684) @LukasTy
```

Reviewer 3: https://github.com/mui/mui-x/pull/5761#discussion_r943409743

I prefer mine :D

> - [pickers] fix to trigger `onChange` when clearing or accepting `Invalid date` (#5740) @LukasTy

Reviewer 3: ```suggestion
- [pickers] Trigger `onChange` when clearing or accepting `Invalid date` (#5740) @LukasTy
```

### Docs

- [docs] Add RFC GH issue template (#5739) @bytasv

> - [docs] Add description to the GridExportStateParams page (#5654) @oliviertassinari

Reviewer 3: ```suggestion
- [docs] Add description to the `GridExportStateParams` page (#5654) @oliviertassinari
```

- [docs] Improve the Events doc page (#5413) @flaviendelangle
- [docs] Use new edit API in the introduction demos (#5728) @oliviertassinari

### Core

> - [core] Remove duplicated FUNDING.yml file (#5656) @oliviertassinari

Reviewer 4: ```suggestion
- [core] Remove duplicated `FUNDING.yml` file (#5656) @oliviertassinari
```

We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:

- ✨ Improve quick filtering with row grouping (#5701) @alexfauquette
- 📚 Documentation improvements
- 🐞 Bugfixes

### `@mui/x-data-grid@5.15.2` / `@mui/x-data-grid-pro@5.15.2` / `@mui/x-data-grid-premium@5.15.2`

#### Changes

- [DataGrid] Catch errors if rows freezing is not supported (#5711) @cherniavskii
- [DataGrid] Preserve cell mode when entering edit mode while commiting (#5686) @m4theushw
- [DataGridPremium] Let quick filter search in row grouping children (#5701) @alexfauquette

### `@mui/x-date-pickers@v5.0.0-beta.5` / `@mui/x-date-picker-pro@5.0.0-beta.5`

#### Changes

- [TimePicker] Set clock focus outline to `none` (#5758) @LukasTy
- [pickers] Fix theme augmentation with TypeScript (#5596) @alexfauquette
- [pickers] Reset input value when locale is modified (#5310) @alexfauquette
- [pickers] Support `disableHighlightToday` on `MonthPicker` and `YearPicker` (#5562) @flaviendelangle
- [pickers] Fallback to desktop mode when `matchMedia` is unavailable (#5684) @LukasTy
- [pickers] Trigger `onChange` when clearing or accepting `Invalid date` (#5740) @LukasTy

> 

Reviewer 5: ```suggestion
- [pickers] Add `react-dom` to peerDependencies (#5752) @cherniavskii

```

Author: I've added it as first change, because the script seems to list those in descending order. 😉

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 5.15.2

_Aug 11, 2022_

We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:

- ✨ Improve quick filtering with row grouping (#5701) @alexfauquette
- 📚 Documentation improvements
- 🐞 Bugfixes

### `@mui/x-data-grid@5.15.2` / `@mui/x-data-grid-pro@5.15.2` / `@mui/x-data-grid-premium@5.15.2`

#### Changes

- [DataGrid] Catch errors if rows freezing is not supported (#5711) @cherniavskii
- [DataGrid] Preserve cell mode when entering edit mode while commiting (#5686) @m4theushw
- [DataGridPremium] Let quick filter search in row grouping children (#5701) @alexfauquette

### `@mui/x-date-pickers@v5.0.0-beta.5` / `@mui/x-date-picker-pro@5.0.0-beta.5`

#### Changes

> - [pickers] Add `react-dom` to peerDependencies (#5752) @cherniavskii
> - [TimePicker] Set clock focus outline to `none` (#5758) @LukasTy

Reviewer: Only for esthetic reasons 

```suggestion
- [TimePicker] Set clock focus outline to `none` (#5758) @LukasTy
- [pickers] Add `react-dom` to peerDependencies (#5752) @cherniavskii
```

Author: You are suggesting a change of order?
The order is in line with what would be produced by our script generating changelog. 🤔 
Are you sure we need this?
